test/rgb_matrix/animation_test.exs

```diff
@@ -1,12 +1,190 @@
 defmodule RGBMatrix.AnimationTest do
   use ExUnit.Case
 
+  alias Layout.LED
   alias RGBMatrix.Animation
+  alias RGBMatrix.Animation.Config.FieldType.{Integer, Option}
+
+  setup_all [
+    :create_leds,
+    :create_frame,
+    :create_animation_module,
+    :create_config,
+    :create_animation,
+    :create_schema
+  ]
 
   test "can get an animation type's human-readable name" do
-    defmodule MockAnimation.HueWave do
+    assert Animation.type_name(MockHueWave) == "Mock Hue Wave"
+    assert Animation.type_name(MockAnimations.Mock.HueWave) == "Hue Wave"
+    assert Animation.type_name(Mock.Animations.Hue.Wave) == "Wave"
```

Comment:
⚠️ (recommended)

I recommend putting this back to the assertion it was before, as these changes are locking in behavior we _don't_ want, or they're redundant. The purpose of the original test was to make sure things don't break with the animation name until we get to the better design of an animation providing it's own human-readable name, like with a `name "Hue Wave"` macro. The previous test should have accounted for that without breaking.

```ex
assert Animation.type_name(MockAnimation.HueWave) == "Hue Wave"
```

Reply:
Yeah, good point.